src/test/java/com/pubnub/contract/access/step/GivenSteps.kt

```diff
@@ -72,33 +75,82 @@ class GivenSteps(private val grantTokenState: GrantTokenState) {
         grant_permission(permissionType)
     }
 
+    //DELETE http://localhost:8090/v3/pam/sub-c-067a9e98-fb50-11eb-bc10-56aa1f324334/grant/A8TlaF6egF9dE6a0DzVpnEyftEt7Nl?pnsdk=PubNub-Kotlin%2F6.2.0&requestid=6cc73a57-d282-4685-bef1-7448895b8821&uuid=pn-6ecefeb0-4625-4dc2-b2da-ffea6c784678
```

Comment:
@kleewho some keys leaking here in comments

Reply:
I'll remove them